### PR TITLE
chore(llmobs): migrate span payload structure to meet new evp updates

### PR DESCRIPTION
**Pull request by Yun-Kim**
This PR updates the structure of LLMObs span payloads to meet the new LLMObs evp public API requirements.

Existing tests should cover this change, and this is a completely internal change meaning there are no public-facing changes.

## Checklist

- [x] Change(s) are motivated and described in the PR description
- [x] Testing strategy is described if automated tests are not included in the PR
- [x] Risks are described (performance impact, potential for breakage, maintainability)
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed or label `changelog/no-changelog` is set
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/))
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))
- [x] If this PR changes the public interface, I've notified `@DataDog/apm-tees`.

## Reviewer Checklist

- [x] Title is accurate
- [x] All changes are related to the pull request's stated goal
- [x] Description motivates each change
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes
- [x] Testing strategy adequately addresses listed risks
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] Release note makes sense to a user of the library
- [x] Author has acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

File: ddtrace/llmobs/_writer.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def enqueue(self, event: LLMObsSpanEvent) -> None:
         self._enqueue(event)
 
     def _data(self, events: List[LLMObsSpanEvent]) -> Dict[str, Any]:
-        return {"ml_obs": {"stage": "raw", "type": "span", "spans": events}}
+        return {"_dd.stage": "raw", "event_type": "span", "spans": events}
```
Comment (sabrenner) [inline ddtrace/llmobs/_writer.py:142]: we'll need a change for eval metrics too, right? ie `"event_type": "eval-metric"` in `LLMObsEvalMetricWriter._data`? or will this be addressed in a separate PR?

**Review by sabrenner — APPROVED** [review]
lgtm for submitting spans with new evp structure!

File: ddtrace/llmobs/_writer.py (hunk 1 above)
Comment (Yun-Kim) [reply ddtrace/llmobs/_writer.py:142]: We won't need to change eval metrics, since evaluation metrics are submitted to the Rapid GO API (this payload structure is only regarding our EVP endpoint). But yes, we'll need to change the eval metric payloads once we change eval metrics over to use the public EVP endpoint
